### DISCUSSION TITLE
onChange with v-model issue

### DISCUSSION THREAD

**Discussion by 7iomka** [vuejs/petite-vue#65] [Q&A]
Hi.
https://codepen.io/7iomka/pen/GRmbOrZ?editors=1111
My question is - why `this.prop` have old not updated value in @change callback?

**Comment by 7iomka** [comment]
I vue prop used for v-model updates before @change event
https://codesandbox.io/s/interesting-brook-5nw7m?file=/src/App.vue
**This is a very serious discrepancy in the logic of work!**

**Comment by mxpshn** [comment]
@7iomka  I managed to get the desired behaviour in two steps: 

1. change `v-model` to `v-model.lazy` 
2. write `v-model.lazy` before `@change` attribute

<img width="685" alt="Screenshot 2021-08-25 at 15 10 44" src="https://user-images.githubusercontent.com/620034/130788580-bbf160d7-97e3-4c4e-ad31-29c8c6817449.png">

**Comment by Alexufo** [comment]
@maxbublik what sense of v-model in vue petite? We cant watch on prop value, and do something after it change. v-model totally useless.
@change event can only emit selected value.


PS. is it equal?  

 @change="handleChange($event)  // e.target.value
@change="handleChange($el.value)" ?
